.github/workflows/check.yml

```diff
@@ -0,0 +1,167 @@
+name: Check
```

Comment:
This division is bad. None of the actions below check if the JS builds. The JS step can be tested only after WASM binary is generated, so we need the full "build" command here. How would it change the times if we would run `run build` next to scripts below? If it will change the times drastically we can live without JS being tested on CI - the chance it will not work are so small that wecan ignore them (it will onlyc check if dependencies resolve because js code is uncheckable).